try:
    x = int(input("Enter a integer : "))
    print ("x = ", x)
except:
    print ("Something went worong...!")
finally:

try:
    print("End of try except")try:
    x = int(input("Enter a integer : "))
    print ("x = ", x)
except:
    print ("Something went worong...!")
finally:
    print("End of try except")